<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" total="9" passed="6">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="114004" started-at="2016-01-14T01:22:14Z" finished-at="2016-01-14T01:24:08Z">
    <groups>
      <group name="SmokeTest">
        <method signature="Login_Scenario.avalid_login(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:sceanrios_TCs.Login_Scenario@108b2d7]" name="avalid_login" class="sceanrios_TCs.Login_Scenario"/>
      </group> <!-- SmokeTest -->
    </groups>
    <test name="Test" duration-ms="114004" started-at="2016-01-14T01:22:14Z" finished-at="2016-01-14T01:24:08Z">
      <class name="sceanrios_TCs.Cart_Scenario">
        <test-method status="PASS" signature="addcart(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:sceanrios_TCs.Cart_Scenario@1690630]" name="addcart" duration-ms="19709" started-at="2016-01-14T06:52:15Z" data-provider="DP_addcart" finished-at="2016-01-14T06:52:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[GB_12]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1.0]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[qtpworld2008@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[learn123]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Flood and Fire]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[1.0]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[Flood and Fire]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addcart -->
        <test-method status="PASS" signature="deletecart(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:sceanrios_TCs.Cart_Scenario@1690630]" name="deletecart" duration-ms="15783" started-at="2016-01-14T06:52:35Z" data-provider="DP_deletecart" finished-at="2016-01-14T06:52:51Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[GB_15]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1.0]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[qtpworld2008@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[learn123]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Flood and Fire]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[1.0]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[Your shopping cart is empty.]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deletecart -->
      </class> <!-- sceanrios_TCs.Cart_Scenario -->
      <class name="sceanrios_TCs.Login_Scenario">
        <test-method status="PASS" signature="avalid_login(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:sceanrios_TCs.Login_Scenario@108b2d7]" name="avalid_login" duration-ms="10705" started-at="2016-01-14T06:53:37Z" data-provider="DP_valid_login" finished-at="2016-01-14T06:53:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[GB_07]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1.0]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[qtpworld2008@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[learn123]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[qtpworld2008 g]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- avalid_login -->
        <test-method status="PASS" signature="invalid_login(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:sceanrios_TCs.Login_Scenario@108b2d7]" name="invalid_login" duration-ms="10858" started-at="2016-01-14T06:53:48Z" data-provider="DP_invalid_login" finished-at="2016-01-14T06:53:59Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[GB_01]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1.0]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Raje]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[ghagh]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Sorry we were unable to complete this step because :]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalid_login -->
        <test-method status="PASS" signature="invalid_login(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:sceanrios_TCs.Login_Scenario@108b2d7]" name="invalid_login" duration-ms="9640" started-at="2016-01-14T06:53:59Z" data-provider="DP_invalid_login" finished-at="2016-01-14T06:54:08Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[GB_02]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1.0]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Raje]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[learn123]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Sorry we were unable to complete this step because :]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalid_login -->
      </class> <!-- sceanrios_TCs.Login_Scenario -->
      <class name="sceanrios_TCs.Search_Scenario">
        <test-method status="PASS" signature="invalidSearch(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:sceanrios_TCs.Search_Scenario@6776ad]" name="invalidSearch" duration-ms="11725" started-at="2016-01-14T06:52:51Z" data-provider="DP_invalid_search" finished-at="2016-01-14T06:53:02Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[GB_08]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1.0]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[bbsj54sfdds]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Sorry..'bbsj54sfdds' does not match any product on Rediff Books.]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidSearch -->
        <test-method status="FAIL" signature="validSearch(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:sceanrios_TCs.Search_Scenario@6776ad]" name="validSearch" duration-ms="12564" started-at="2016-01-14T06:53:02Z" data-provider="DP_valid_search" finished-at="2016-01-14T06:53:15Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[GB_09]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1.0]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Ways]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[495.0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
	The actual found is 491 The expected is 495 expected [495] but found [491]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
	The actual found is 491 The expected is 495 expected [495] but found [491]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at sceanrios_TCs.Search_Scenario.validSearch(Unknown Source)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.TestNG.privateMain(TestNG.java:1355)
	at org.testng.TestNG.main(TestNG.java:1324)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validSearch -->
        <test-method status="FAIL" signature="validSearch(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:sceanrios_TCs.Search_Scenario@6776ad]" name="validSearch" duration-ms="12064" started-at="2016-01-14T06:53:15Z" data-provider="DP_valid_search" finished-at="2016-01-14T06:53:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[GB_10]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1.0]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Chandler]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[428.0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
	The actual found is 414 The expected is 428 expected [428] but found [414]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
	The actual found is 414 The expected is 428 expected [428] but found [414]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at sceanrios_TCs.Search_Scenario.validSearch(Unknown Source)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.TestNG.privateMain(TestNG.java:1355)
	at org.testng.TestNG.main(TestNG.java:1324)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validSearch -->
        <test-method status="FAIL" signature="validSearch(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:sceanrios_TCs.Search_Scenario@6776ad]" name="validSearch" duration-ms="10044" started-at="2016-01-14T06:53:27Z" data-provider="DP_valid_search" finished-at="2016-01-14T06:53:37Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[GB_11]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1.0]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Jaico]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[4.0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
	The actual found is 6 The expected is 4 expected [4] but found [6]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
	The actual found is 6 The expected is 4 expected [4] but found [6]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at sceanrios_TCs.Search_Scenario.validSearch(Unknown Source)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.TestNG.privateMain(TestNG.java:1355)
	at org.testng.TestNG.main(TestNG.java:1324)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validSearch -->
      </class> <!-- sceanrios_TCs.Search_Scenario -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
